code/modules/loadout/categories/heads.dm

```diff
@@ -132,6 +132,30 @@
 	name = "Rose"
 	item_path = /obj/item/food/grown/rose
 
+/datum/loadout_item/head/sunflower
+	name = "Sunflower"
+	item_path = /obj/item/food/grown/sunflower
+
+/datum/loadout_item/head/poppy
+	name = "Poppy"
+	item_path = /obj/item/food/grown/poppy
+
+/datum/loadout_item/head/lily
+	name = "Lily"
+	item_path = /obj/item/food/grown/poppy/lily
+
+/datum/loadout_item/head/geranium
+	name = "Geranium"
+	item_path = /obj/item/food/grown/poppy/geranium
+
+/datum/loadout_item/head/fraxinella
+	name = "Fraxinella"
+	item_path = /obj/item/food/grown/poppy/geranium/fraxinella
```

Comment:
I'd be somewhat concerned about offering Fraxinella in loadouts, due to being mutatation-only and offering useful chems for crossbreeding since it has some real utility where most of the other flowers are very mild in their effects. Technically any of these offers an advantage of being able to be made into seeds, but this is the only one that I feel like has any real impact.

Reply:
I'll be honest, I have no idea about the chems, so if it's a problem, I can remove them.

Reply:
why havent you

Reply:
i was not given a concrete answer on whether to remove them or not, just that ArcaneMusic would be "somewhat concerned" about them